device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-A96C8V8/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,46 @@
+
+{#
+    Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES.
+    Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '5m' %}
+{%-set ports2cable = {
+            'torrouter_server'       : '5m',
+            'leafrouter_torrouter'   : '300m',
+            'spinerouter_leafrouter' : '1500m'
+       }
+-%}
+{% set ingress_lossless_pool_size =  '46415808' %}
```

Comment:
as a convention, the pool size needs to be rounded down to the cell size boundary for `egress_lossless_pool_size` only.
for other pools, we do not round them down.

Reply:
Updated